src/containers/NavBar/NavBar.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+
+import React, { Component } from 'react';
+import { Link, withRouter } from 'react-router-dom';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import IconButton from '@material-ui/core/IconButton';
+import Badge from '@material-ui/core/Badge';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import NotificationsIcon from '@material-ui/icons/Notifications';
+import MenuItem from '@material-ui/core/MenuItem';
+import Menu from '@material-ui/core/Menu';
+import MenuIcon from '@material-ui/icons/Menu';
+import Drawer from '@material-ui/core/Drawer';
+
+import MenuList from './MenuList';
+
+import './NavBar.css';
+
+type Props = {
+  title: string,
+  notificationNumber: number,
+  userId: number,
+  history: Function,
+  logoutCallback: Function,
+};
+
+type State = {
+  openProfile: boolean,
+  openDrawer: boolean,
+};
+
+class NavBar extends Component<Props, State> {
+  constructor() {
+    super();
+    this.state = {
+      openProfile: false,
+      openDrawer: false,
+    };
+  }
+
+  handleOpen = (open, value) => {
+    this.setState(prevState => ({
```

Comment:
pourquoi prevState ? dans ce cas la tu peux juste faire` this.setState({[open]: value})`